db/migrations/state/0013_test.go

```diff
@@ -0,0 +1,95 @@
+package migrations_test
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// this migration changes length of the token name
+type migrationTest0013 struct{}
+
+func (m migrationTest0013) insertBlock(blockNumber uint64, db *sql.DB) error {
+	const addBlock = "INSERT INTO state.block (block_num, received_at, block_hash) VALUES ($1, $2, $3)"
+	if _, err := db.Exec(addBlock, blockNumber, time.Now(), "0x29e885edaf8e4b51e1d2e05f9da28161d2fb4f6b1d53827d9b80a23cf2d7d9f1"); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (m migrationTest0013) insertRowInOldTable(db *sql.DB, args []interface{}) error {
+	insert := `
+        INSERT INTO state.exit_root (block_num, timestamp, mainnet_exit_root, rollup_exit_root, global_exit_root) 
+                             VALUES ($1,        $2,        $3,                $4,               $5  );`
+
+	_, err := db.Exec(insert, args...)
+	return err
+}
+
+func (m migrationTest0013) InsertData(db *sql.DB) error {
+	var err error
+	if err = m.insertBlock(uint64(123), db); err != nil {
+		return err
+	}
+	if err = m.insertBlock(uint64(124), db); err != nil {
+		return err
+	}
+	if err = m.insertRowInOldTable(db, []interface{}{123, time.Now(), "mer", "rer", "ger"}); err != nil {
```

Comment:
what about using real mock values instead mer, rer and so on? For instance `0x29e885edaf8e4b51e1d2e05f9da28161d2fb4f6b1d53827d9b80a23cf2d7d9f1`